### ISSUE TITLE
Improve RegSet to be able to have 128 registers / Define all FP and V registers

### ISSUE THREAD

**Issue by gut** [PPC64/hhvm#80]
Currently there is the limitation of indexing only up to 64 registers in HHVm. If we improve it to 128 registers (with #ifdef's), then we could add the rest of the Floating Point and Vector registers.

**Comment by gut** [comment]
Max Wang already implemented this, now we have to change our settings to make use of it.
See commit https://github.com/facebook/hhvm/commit/cb93af7ab58082f7664dcb1a7e7bc0e8d56cc6f4

**Comment by gut** [comment]
Related task: #11

**Comment by racardoso** [comment]
I think we can close this issue.

**Comment by gut** [comment]
I don't think so. We still need to define more registers on our architecture:
https://github.com/PPC64/hhvm/blob/next/hphp/ppc64-asm/asm-ppc64.h#L396
